src/OrchardCore.Modules/OrchardCore.Settings/SiteSettings.cs

```diff
@@ -28,5 +28,6 @@ public class SiteSettings : Entity, ISite
         public string CdnBaseUrl { get; set; }
         public RouteValueDictionary HomeRoute { get; set; } = new RouteValueDictionary();
         public bool AppendVersion { get; set; } = true;
+        public string Meta { get; set; }
     }
```

Comment:
This should be added to the `ISite` interface as well.

And then an update to both the deployment step and the content step is required with the matching property.
(see the comment at the top of the file)